templates/cluster-template-ipv6.yaml

```diff
@@ -197,6 +198,343 @@ spec:
   tenantID: ${AZURE_TENANT_ID}
   type: ServicePrincipal
 ---
+apiVersion: addons.cluster.x-k8s.io/v1beta1
+kind: ClusterResourceSet
```

Comment:
Note: ClusterResourceSet is still behind a feature flag which means this change will make enabling the feature flag required to use _all_ release flavors after they are published as artifacts for the next release. We need big warning signs in the release notes and we should update the quickstart and all docs to warn about this.

Reply:
quickstart and docs updated